components/generic/Icons/FavouriteIconFilled.tsx

```diff
@@ -0,0 +1,29 @@
+import { clsx as cx } from 'clsx';
+import { DEFAULT_HEIGHT, DEFAULT_WIDTH } from './constants';
+
+type FavouriteIconFilledProps = {
+  width?: number;
+  height?: number;
+  className?: string;
+};
+
+export const FavouriteFilledIcon = ({
+  width = DEFAULT_WIDTH,
+  height = DEFAULT_HEIGHT,
+  className,
+}: FavouriteIconFilledProps) => {
+  return (
+    <svg
+      className={cx('fill-pink-500', className)}
```

Comment:
Pink?

Reply:
I think it looks better than shades of red.

Reply:
I would use `amber` just like on the design.